### ISSUE TITLE
SW-100031

### ISSUE THREAD

**Issue by aizaz-clym** [aizaz-clym/clym-test#6]
## Structural markup should be used for presentational effect

| Type | Standard | Category | Occurrences | Page | Scanned | Status |
| --- | --- | --- | --- | --- | --- | --- |
| Automatic | WCAG | Common Failures | 1 | Home | 2026-07-15 | 🔴 Failed |

### Steps

1. Check that the element's semantic meaning is exposed to assistive technology and appropriate for the content of the element.

### Criteria

| Criteria | Name | Description | Level |
| --- | --- | --- | --- |
| [1.3.1](https://www.w3.org/WAI/WCAG22/quickref#info-and-relationships) | Info and Relationships | Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text. | A |

### Selector

`html`

### HTML

```html
<html lang="en">
```

---

> Created automatically by [Clym](https://clym.io)
